index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Youtube clone</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <!-- I suggest we use the flex displaying to better handle the direction and sizes of boxes -->
+    <header>
+    <nav>
+      <!-- In the navbar, we first have image, the first is the 3 bands and the secon is the logo -->
+      <div class="logo-container">
+        <a href="#subscribe"><img src="./img/humburgerIcon.png" alt="Menu icon" class="menu"/></a>
+        <img src="./img//youtube1.png" alt="youtube logo" class="logo" />
+      </div>
+      <input
+        class="search"
+        type="text"
+        name="search"
+        placeholder="Search"
+      /><span><button class="btn-search">Search</button></span>
+      <div class="btn-section">
+        <div style="width: 50px"></div>
```

Comment:
This is an inline style attribute. search for this pattern, `style="... some properties..." `, remove it from the HTML file and use the stylesheet to style the affected elementes.

Reply:
Thank you dear for this feedback, all the issues mentioned are now fixed. Please take a look again